src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
it is for most of the current consumers, but i assumed that since it's no longer the recommended configuration method, we're still supporting the old method, making it backwards compatible. words are hard.